src/test/resources/artifacts/ESB/connector/config/cerner.properties

```diff
@@ -0,0 +1,42 @@
+proxyDirectoryRelativePath=/artifacts/ESB/config/proxies/cerner/
+requestDirectoryRelativePath=/artifacts/ESB/config/restRequests/cerner/
+base=https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca
```

Comment:
Move properties that are specific to accounts to  separate properties file inside "connector/repository/" folder .
Refer [1].

[1]https://github.com/wso2-extensions/esb-connector-googlespreadsheet/pull/13/files#diff-224e68342d61e85c9386bf00d3afe9dfR1